.github/workflows/run-integration-tests.yaml

```
@@ -0,0 +1,35 @@
+name: Run integration tests
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      image_name:
+        description: The image repository and name to use.
+        required: false
+        default: ghcr.io/fluent/fluent-bit/master
+        type: string
+      image_tag:
+        description: The image tag to use.
+        required: false
+        default: x86_64
+        type: string
+      ref:
+        description: The commit, tag or branch of this repository to checkout
+        type: string
+        required: false
+        default: main
+jobs:
+  run-tests:
+    name: Run integration tests
+    uses: fluent/fluent-bit-ci/.github/workflows/call-run-integration-test.yaml@main
```

Probably use local ref only rather than have to force it always to use the `main` one - then you can run on a branch easily now.
```suggestion
./.github/workflows/call-run-integration-test.yaml
```